jsFiles/design.js

```diff
@@ -75,3 +75,79 @@ function skyColor(sunrise, sunset, offsetTime) {
   }
   // end sunset
 }
+
+/**
+ * This function changes the diamensions of the progress bar according
+ * to the sunrise and sunset time, dividing it into three differently
+ * sized parts. It changes the filling of the bar, according to the current time,
+ * and it controls the display on the day light label above the bar
+ * @param {number} sunrise - time stamp with time info
+ * @param {number} sunset - time stamp with time info
+ * @param {number} offsetTime - offset time refering to the time zone
+ */
+function dayProgressBar(sunrise, sunset, offsetTime) {
+  // A day has 1440 min: 60*24
+  const timeInMinutes = timeConversion(
+    currentDateUserLocation,
+    offsetTime,
+    true
+  );
+
+  const sunriseInMinute = timeConversion(sunrise, offsetTime, true);
+  const sunsetInMinutes = timeConversion(sunset, offsetTime, true);
+  const timeInMinDay = sunsetInMinutes - sunriseInMinute;
+
+  const timeInMinAfterSunset = 1440 - sunsetInMinutes;
```

Comment:
I think you can put it in a constant: `const minsInDay = 60 * 24;` then it's a bit easier to read the code.